src/common/state/table.cc

```diff
@@ -8,6 +8,8 @@ static const RetryInfo default_retry = {.num_retries = -1,
                                         .timeout = 10000,
                                         .fail_callback = NULL};
 
+static int64_t next_dummy_timer_id = 1000;
```

Comment:
This might cause problems once we naturally hit timer IDs of 1000 (this will happen naturally because every time a heartbeat is sent we restart the heartbeat timer, which I think increments the timer ID counter).